pandas/errors/__init__.py

```diff
@@ -599,19 +599,30 @@ class IndexingError(Exception):
     """
     Exception is raised when trying to index and there is a mismatch in dimensions.
 
+    Raised by properties like :attr:`.pandas.DataFrame.iloc` when
+    an indexer is out of bounds or :attr:`.pandas.DataFrame.loc` when its index is
+    unalignable to the frame index.
+
+    See Also
+    --------
+    :attr:`.pandas.DataFrame.iloc` : Purely integer-location based indexing for \
```

Comment:
I don't think in the `See Also` section we need those `:attr:` markers, I think just `DataFrame.iloc` should render correctly. Same for the other cases in this PR.

Reply:
Hi Marc, 

Thanks! I pushed these changes up.